### DISCUSSION TITLE
.slp files created during inference

### DISCUSSION THREAD

**Discussion by carson2stoker** [talmolab/sleap#915] [Help!] [answered]
Hi, I'm trying to import my data into SIMBA for action recognition. They told me I need to import the .slp files generated during inference. I didn't know sleap did this and can't seem to locate those files. I've run inference and gotten predictions on my entire video clip. The only .slp files I see are the project ones though. Am I missing something here? Thanks.

**Comment by roomrys** [comment, marked as answer]
Hi @carson2stoker,

In the same directory containing your original `.slp` file, there should be a directory named "predictions" which contains `.predictions.slp` files for all inference runs. Are you able to locate this directory?

Thanks,
Liezl

**Reply by carson2stoker** [reply]
Ah perfect thank you! I didn't realize that those prediction files were .slp. I have it working now.
